bootstrap/absolute.sh

```diff
@@ -17,7 +17,10 @@ set_path_env $(bootstrap_command_path)
 set_path_env $(absolute_path)/node_modules/.bin
 
 # NPM update
-npm update
+if [ ! .pkg_timestamp -nt package.json ]; then
```

Comment:
Actually, I had to consider the case even .pkg_timestamp file is not created, so I wrote code as above.  |if [ package.json -nt .pkg_timestamp ]; goes to false when .pkg_timestamp file doesn't existed. :-)